diff --git a/node_modules/expo-modules-core/android/build.gradle b/node_modules/expo-modules-core/android/build.gradle
index 3772140..bbb85c0 100644
--- a/node_modules/expo-modules-core/android/build.gradle
+++ b/node_modules/expo-modules-core/android/build.gradle
@@ -108,7 +108,8 @@ android {
           "-DREACT_NATIVE_DIR=${expoModuleExtension.reactNativeDir}",
           "-DREACT_NATIVE_TARGET_VERSION=${expoModuleExtension.reactNativeVersion.minor}",
           "-DUSE_HERMES=${USE_HERMES}",
-          "-DUNIT_TEST=${isExpoModulesCoreTests}"
+          "-DUNIT_TEST=${isExpoModulesCoreTests}",
+          "-DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON"
         ]
 
         if (reactNativeWorkletsRootDir != null) {
